packages/replay/package.json

```
@@ -46,7 +46,7 @@
   "homepage": "https://docs.sentry.io/platforms/javascript/session-replay/",
   "devDependencies": {
     "@babel/core": "^7.17.5",
-    "@sentry-internal/rrweb": "1.101.2",
+    "@sentry-internal/rrweb": "^1.102.0",
```

m: Let's keep pinning this to an exact version:

```suggestion
    "@sentry-internal/rrweb": "1.102.0",
```